CHANGELOG.md

```diff
@@ -68,6 +68,7 @@
 - Fixed an issue in `operator-sdk olm-catalog gen-csv` where the generated CSV is missing the expected set of owned CRDs. ([#2017](https://github.com/operator-framework/operator-sdk/pull/2017))
 - The command `operator-sdk olm-catalog gen-csv --csv-version=<version> --update-crds` would fail to copy over CRD manifests into `deploy/olm-catalog` for manifests whose name didn't end with a `_crd.yaml` suffix. This has been fixed so `gen-csv` now copies all CRD manifests specified by `deploy/olm-catalog/csv_config.yaml` by checking the type of the manifest rather than the filename suffix. ([#2015](https://github.com/operator-framework/operator-sdk/pull/2015))
 - Added missing `jmespath` dependency to Ansible-based Operator .travis.yml file template. ([#2027](https://github.com/operator-framework/operator-sdk/pull/2027))
+- Move `t.Fatalf` in sub-goroutine into main-goroutine in function `TestCreateChart`, to make it really take effect when triggered. ([1957](https://github.com/operator-framework/operator-sdk/pull/1957))
```

Comment:
```suggestion

```
Sorry @beautytiger, 

I look it better and actually we should not add the CHANGELOG since it will NOT fix any behaviour for the users. Am I right? Does it only fix the Tests charts?

Reply:
Yes, you are right, i will revert this.